community/community.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: single
+author_profile: false
+title: Zarr Community
+permalink: /community/
```

Comment:
This might not be needed if the file's name were "index.md". Do you mind giving that a try?

Reply:
After removing `layout: single` and renaming the file `index.md` the sidebar disappears. I think we need this.